File: Core python/Assignment/Assignment 2/q10.py
# 10. Write a program to reverse three-digit number.

# Find the sum of three-digit number.

# take 3 digit input number
n = int(input("enter three digit number : "))

# print number before reverse
print(f'original number is : {n}')

# calculate last digit after iteration
d1 = n % 10
n = n // 10

d2 = n % 10
n = n // 10   # give as a last digit

d3 = n % 10

# calculate reverse number
r = (d1 * 100) + (d2 * 10) + (d3)

# print number after reverse
print(f'reverse number is : {r}')

File: C to Java/Assignments/Assignment 3/q10.c
/* Find Sum of first and last digit of given number.
Input: n = 12345
Output: 6 (1 + 5)
*/


void main() {
	
	unsigned long long int  n , firstDigit , lastDigit , sum , digit;
	printf("enter number : ");
	scanf("%llu", &n);
	
	lastDigit = n % 10 ;
	
	while(n > 0) {
		digit = n % 10 ;
		n = n / 10 ;
	}
	
	firstDigit = digit ;
	sum = firstDigit + lastDigit ;
	
	printf("Sum of first and last digit : %d" , sum);
	
	
}